$Id$
Excluded bessel tests as they do not compile. 
FIXME: needs some asm rework perhaps.
--- ltp-full-20061121.orig/testcases/misc/math/float/Makefile	2006-11-21 21:31:27.000000000 +0100
+++ ltp-full-20061121/testcases/misc/math/float/Makefile	2006-12-06 11:31:45.000000000 +0100
@@ -1,10 +1,10 @@
 CFLAGS+= -I../../../../include -Wall -g 
 LOADLIBES+= -lpthread -lm -L../../../../lib -lltp
 
-SRCS=$(wildcard float*.c)
+SRCS=$(patsubst float_bessel.c,,$(wildcard float*.c))
 TARGETS=$(patsubst %.c,%,$(SRCS))
 
-SUBDIRS=bessel exp_log iperb power trigo
+SUBDIRS=exp_log iperb power trigo
 
 all: $(TARGETS)
 	@set -e; for i in $(SUBDIRS); do $(MAKE) -C $$i ; done
